docs/hosting-a-server-with-northstar/dedicated-server/README.md

```diff
@@ -130,6 +130,7 @@ They allow the server admin to set server's properties like the name, TCP port,
 | `ns_private_match_last_mode`                     | Forces the lobby to a specific Gamemode                                                                                                                                                     | `tdm`                          | Any [Gamemode](./#gamemodes)                       |
 | `ns_private_match_last_map`                      | Forces the lobby to a specific map                                                                                                                                                          | `mp_forwardbase_kodai`         | Any [Map](./#maps)                                 |
 | `ns_disallowed_weapons`                          | Blacklists weapons                                                                                                                                                                          |                                | List of [Weapons](./#weapons) separated by a comma |
+| `ns_allow_kill_commands`                     | Enables the use of `kill` in the console           | `0`           | `0-1`                                                        |
```

Comment:
Any chance you could get the indention to match up?
(Kinda a pain to do via GitHub suggestions)

Reply:
yeah, I'll work on this rq

Reply:
hopefully thats better, it matches up on my end

Reply:
Still scuffed for me, trying to fix it on my end now ^^

Reply:
Managed to fix in 2bcc6096c0929cfcbd370b538fcbe6ebe4de9055 ^^